kernel/kernel-api/src/main/java/io/delta/kernel/internal/actions/Protocol.java

```diff
@@ -45,6 +45,24 @@ public class Protocol {
   /// Public static variables and methods                                                       ///
   /////////////////////////////////////////////////////////////////////////////////////////////////
 
+  /**
+   * Helper method to get the Protocol from the row representation.
+   *
+   * @param row Row representation of the Protocol.
+   * @return the Protocol object
+   */
+  public static Protocol fromRow(Row row) {
```

Comment:
nit: use `requireNonNull` to assert that row isn't null before we access `row.issNullAt` etc

Reply:
Done.